apiv2/database/models/tomogram_voxel_spacing.py

```diff
@@ -50,6 +50,7 @@ class TomogramVoxelSpacing(Base):
         back_populates="tomogram_voxel_spacing",
         uselist=True,
         foreign_keys="Tomogram.tomogram_voxel_spacing_id",
+        cascade="all, delete-orphan",
```

Comment:
Should this also delete the orphaned annotation files?

Reply:
Ooh, good catch! updated!